packages/@okta/vuepress-site/docs/reference/core-okta-api/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Core Okta API
+meta:
+  - name: description
+    content: Learn how the Okta API works and learn about the compatibility rules and design principles.
+---
+# Core Okta API
+
+The Core Okta API is the primary way that apps and services interact with Okta. You can use it to implement basic auth functions such as signing in your users and programmatically managing your Okta objects. 
+
+## Sign in Your Users 
+API endpoints to authenticate your users, challenge for factors, recover passwords, and more. For example:
+  - The [Authentication API](/docs/reference/api/authn) controls user access to Okta.
+  - The [OpenID Connect & OAuth 2.0 API](/docs/concepts/oauth-openid) controls users access to your applications.
+
+## Manage Okta Objects
+REST endpoints to configure objects whenever you need. For example:
+- The [Apps API](/docs/reference/api/apps/) is used to manage Apps and their association with Users and Groups.
+- The [Users API](/docs/reference/api/users) is used for CRUD operations on Users.
+- The [Sessions API](/docs/reference/api/sessions/) creates and manages user's authentication sessions. 
+- The [Policy API](/docs/reference/api/policy/) creates and manages settings such as a user's session lifetime. 
+- The [Factors API](/docs/reference/api/factors/) is used to enroll, manage, and verify factors for multi-factor authentication (MFA). 
+
+<HorizontalDivider/>
```

Comment:
We don't use horizontal dividers anywhere else in the site. Is there a reason that these were added here?

Reply:
As I mentioned before, those were present on mockups, so I've added them. 
![image](https://user-images.githubusercontent.com/68992536/116753656-172dac80-aa10-11eb-9a67-979b0c35342b.png)